src/bors/handlers/mod.rs

```diff
@@ -48,10 +48,10 @@ pub async fn handle_bors_repository_event<Client: RepositoryClient>(
     ctx: Arc<BorsContext>,
 ) -> anyhow::Result<()> {
     let db = Arc::clone(&ctx.db);
-    match event {
-        BorsRepositoryEvent::Comment(comment) => {
-            // We want to ignore comments made by this bot
-            if let Some(repo) = get_repo_state(state, &comment.repository) {
+    if let Some(repo) = get_repo_state(state, &event.repository()) {
```

Comment:
Clippy warning + let's turn this into `let-else` to avoid the nesting:
```rust
let Some(repo) = get_repo_state(..) else {
   log::error!("Received event {event:?} for an unknown repository {}", event.repository());
   return Ok(());
}
```